index.html

```diff
@@ -9,19 +9,21 @@
 </head>
 
 <body>
-    <div class="navbar">
-        <img class="pic1" src="image_1.png" alt="geometry">
-        <img class="menu" src="ic_menu.png" alt="Menu">
-    </div>
+    <header>
+        <a href="index.html">
+            <img class="pic1" src="image_1.png" alt="geometry">
+        </a>
+        <nav><img class="menu" src="ic_menu.png" alt="Menu"></nav>
+    </header>
 
-    <div class="headline">
+    <section class="headline">
         <h1 class="name" >Emilia Zambrano</h1>
         <p class="Presentation" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
             If you like what you see and have a project you need coded, do not hesitate and contact me.
         </p>
         <button class="start" type="button">Start collaboration</button>
         <img class="pic2" src="image_2.png" alt="geometry">
```

Comment:
- [ ] (OPTIONAL) This image here is a decoration and as a result, it would be better to add it as a background image using CSS. You can learn more about background images [here](https://www.w3schools.com/cssref/pr_background-image.asp) 👍

Reply:
@Kingstalux Thank you for your comments! I will dot it!